wire-tests/src/jvmKotlinInteropTest/proto-kotlin/com/squareup/wire/protos/kotlin/simple/SimpleMessage.kt

```diff
@@ -599,18 +600,23 @@ class SimpleMessage(
   }
 
   enum class NestedEnum(
-    override val value: Int
+    override val value: Int,
+    val deprecated: Boolean?
   ) : WireEnum {
-    FOO(1),
+    FOO(1, null),
 
-    BAR(2),
+    BAR(2, null),
 
-    BAZ(3),
+    BAZ(3, null),
 
     @Deprecated(message = "BUZ is deprecated")
-    BUZ(3);
+    BUZ(3, true);
 
     companion object {
+      val ENUM_OPTIONS: EnumOptions = EnumOptions(
```

Comment:
In interop mode we should generate `@JvmField`s for those, let's do this in a followup.

Reply:
sounds good!